SPECS-EXTENDED/tardev-snapshotter/tardev-snapshotter.spec

```diff
@@ -0,0 +1,78 @@
+%global debug_package %{nil}
+
+Summary: Tardev Snapshotter for containerd
+Name: tardev-snapshotter
+Version: 3.2.0.tardev1
+Release: 1%{?dist}
+License: ASL 2.0
+Group: Tools/Container
+# URL: https://www.containerd.io
+Vendor: Microsoft Corporation
+Distribution: Azure Linux
+
+Source0:https://github.com/microsoft/kata-containers/archive/refs/tags/%{version}.tar.gz#/%{name}-%{version}.tar.gz
+# Note: the %%{name}-%%{name}-%%{version}-cargo.tar.gz file contains a cache created by capturing the contents downloaded into $CARGO_HOME.
```

Comment:
We could update the note to use regenerate-archives.sh.

Reply:
note updated